windows/src/desktop/kmshell/xml/config.js

```diff
@@ -130,15 +132,29 @@ document.addEventListener("DOMContentLoaded", windowResize);
       break;
     case 37:    // left
     case 109:   // minus
-      var k = document.getElementById('list_detail_'+n);
-      if( !k ) break;
-      if(k.style.display == 'block') list_detail(event,n);
+      {
+        let keyboard_name = n.substring(9); // remove keyboard_ from element name
+        let kbd_grid = document.getElementById('keyboard_grid_'+keyboard_name);
+        if( !kbd_grid ) break;
+        let kbd_list = document.getElementById('list_'+n);
+        if( !kbd_list ) break;
+        if(!kbd_list.classList.contains('expanded')) break;
+        ///  hide more details for the expanded list
+        if ($(kbd_grid).hasClass('grid_rows_hide')) more_detail(kbd_grid.id);
+      }
       break;
     case 39:    // right
     case 107:   // plus
-      var k = document.getElementById('list_detail_'+n);
-      if( !k ) break;
-      if(!k.style.display || k.style.display == 'none') list_detail(event,n);
+      {
+        let keyboard_name = n.substring(9); // remove keyboard_ from element name
+        let kbd_grid = document.getElementById('keyboard_grid_'+keyboard_name);
+        if( !kbd_grid ) break;
+        let kbd_list = document.getElementById('list_'+n);
+        if( !kbd_list ) break;
+        if(!kbd_list.classList.contains('expanded')) break;
+        ///  show more details for the expanded list
+        if (!$(kbd_grid).hasClass('grid_rows_hide')) more_detail(kbd_grid.id);
```

Comment:
```suggestion
        if (!$(kbd_grid).hasClass('grid_rows_hide')) {
          more_detail(kbd_grid.id);
        }
```

Per style guide, trying to avoid one-liner ifs.